src/utils/plotSpecs/generateViolinSpec.js

```diff
@@ -368,7 +368,7 @@ const generateSpec = (config, plotData) => {
     ],
     title:
     {
-      text: config.title.text || (config.shownGene !== 'notSelected' ? config.shownGene : ''),
```

Comment:
Is the notSelected case not possible anymore?

Reply:
Its more clean when the default value is just null instead of notSelected

Reply:
yes, i changed the default value to be null, instead of this hacky 'notSelected' value